open_spiel/algorithms/cfr.cc

```diff
@@ -15,13 +15,26 @@
 #include "open_spiel/algorithms/cfr.h"
 
 #include <algorithm>
+#include <array>
 
 #include "open_spiel/abseil-cpp/absl/algorithm/container.h"
+#include "open_spiel/abseil-cpp/absl/strings/charconv.h"
+#include "open_spiel/abseil-cpp/absl/strings/numbers.h"
 #include "open_spiel/spiel_utils.h"
+#include "open_spiel/utils/serialization.h"
 
 namespace open_spiel {
 namespace algorithms {
 
+constexpr const int kSerializationVersion = 1;
+constexpr const char* kSerializeMetaSectionHeader = "[Meta]";
+constexpr const char* kSerializeGameSectionHeader = "[Game]";
+constexpr const char* kSerializeSolverTypeSectionHeader = "[SolverType]";
+constexpr const char* kSerializeSolverSpecificStateSectionHeader =
+    "[SolverSpecificState]";
+constexpr const char* kSerializeSolverValuesTableSectionHeader =
+    "[SolverValuesTable]";
+
```

Comment:
Can you add some comments here explaining on a high-level the serialization support in the CFR solvers, and how to enable it?

Reply:
Done.